src/routes/docs/DocsNavTree.svelte

```diff
@@ -1,12 +1,19 @@
 <script lang="ts">
-	import { is_tree } from './documentation.shared';
-	import type { DocsTree } from './documentation.shared';
+	import { is_tree, type DocsTreeNode } from './documentation.shared';
+	import type { DocsTree, DocumentInfo } from './documentation.shared';
 
 	import DocsNavNode from './DocsNavNode.svelte';
 
 	export let tree: DocsTree;
 	// How deeply nested this is.
 	export let nested = 0;
+
+  function assert_info_node(v: DocsTreeNode) {
+    if (!is_tree(v)) {
```

Comment:
Wouldn't this function always be `!is_tree` because of the if-else statement?

Reply:
@Axelen123 help the error you threw is whats not letting me merge gah whys it there

Reply:
lol sorry I forgor.
I will check this in a few hours

Reply:
I think it is best to move it to the docs file. I will do that real quick.

Reply:
The reason why we need the function is because the Array has both types of nodes. We get the nodes from the `each` block in the template but it is not possible to use `as` outside script tags even though we know it is an info node. I added the check just in case someone tries to use it incorrectly.